### DISCUSSION TITLE
Increments doesn't work and patch functuionality either

### DISCUSSION THREAD

**Discussion by arturo-villalpando** [PaulHatch/semantic-version#135] [Q&A]
I will try to implemente the semantic version system, but the increment doesn't work.

Actually I added a "bump_each_commit_patch_pattern", and I created a commit with "patch" word, but doesn't work.

I don't know if I don't have a correct implementation, could you help me with this please.
 
 `- name: Git Semantic Version
      uses: PaulHatch/semantic-version@v5.3.0
      with:
         tag_prefix: "v"
         enable_prerelease_mode: true
         bump_each_commit: true
         bump_each_commit_patch_pattern: "patch"
         major_pattern: "major"
         minor_pattern: "minor"
         version_format: "${major}.${minor}.${patch}-${increment}"`

The error is the next one:

`Run PaulHatch/semantic-version@v5.3.0
  with:
    tag_prefix: v
    enable_prerelease_mode: true
    bump_each_commit: true
    bump_each_commit_patch_pattern: patch
    major_pattern: major
    minor_pattern: minor
    version_format: ${major}.${minor}.${patch}-${increment}
    branch: HEAD
    use_branches: false
    version_from_branch: false
    search_commit_body: false
    user_format_type: csv
    debug: false
  env:
    REGISTRY: registry.digitalocean.com/ctickets-container-registry
    IMAGE_NAME: cm-adverts
Warning: None of the 1 tags(s) found were valid version tags for the present configuration. If this is unexpected, check to ensure that the configuration is correct and matches the tag format you are using.
Version is 0.0.1-0`

The previous versions is "0.0.1-0", then the version doesn't change...

<img width="1463" alt="Captura de pantalla 2024-01-17 a la(s) 18 45 25" src="https://github.com/PaulHatch/semantic-version/assets/58632310/8c39e0f2-44cc-4f2e-9817-79700126e3f3">

**Comment by PaulHatch** [comment]
Hey @arturo-villalpando,

Adding the `bump_each_commit_patch_pattern` will **prevent** the version from incrementing unless the pattern specified is present. I cannot see what commits you have present in your repository so I can't say if you're experiencing issues or if the action is working normally. The warning about the tag not matching indicates that 1 tag was found, but it didn't match the expected pattern, in your case it is expecting tags that resemble `v1.2.3`.

If you can provide more information using one of these three methods, I can take a look: https://github.com/PaulHatch/semantic-version/blob/master/contributing.md#getting-help

Also have a look at the guide for an overview of how different approaches work and how to configure them: https://github.com/PaulHatch/semantic-version/blob/master/guide.md
